### ISSUE TITLE
a11y: Multiple-tab options cannot be accessed with keyboard navigation

### ISSUE THREAD

**Issue by IsaSih** [rancher/dashboard#13260]
**Setup**
- Rancher version: 
Rancher | v2.11-a1bda936c809e0394520c9d40d2998529223401e-head
Dashboard | master 478e195
Rancher CLI | v2.11.0-rc.1

**Describe the bug**
When trying to use keyboard navigation to input values in options across multiple tabs, it's not possible to access the options inside tabs other than the last one, as observed in the screenshot.

**To Reproduce**
1- From the Rancher UI, select a cluster 
2- Hit Project/Namespaces, then select `Create a namespace`
3- Try to add values to options that are located in the first 2 tabs of the multiple-tab component using keyboard navigation

**Result**
User is not able to access the options inside the tabs other than the last in the multiple-tab component.

**Expected Result**
User must be able to access and select all options inside the multiple-tab component.

**Screenshots**

https://github.com/user-attachments/assets/345865d5-b56f-4cfd-9943-bbc4d89505d6

**Comment by aalves08** [comment]
@IsaSih the problem mentioned here is related to the tab content not being directly accessible for keyboard after choosing/activating a tab?

**Comment by izaac** [comment]
Rancher version: `v2.11-0b081e077e830fa17b86151d21c786fb8d07c6de-head`
UI version: `master 68fb044`
Chrome version: `133.0.6943.141`
Firefox version: `135.0.1`

Tabs on cluster creation, k8s objects are now accessible navigating with the keyboard using `Tab` then in that area the Tabs can be navigated with arrow keys. ✅
